docs/guides/guides.md

```diff
@@ -61,3 +61,7 @@ Specifically, this guide will walk through the implementation of the following b
  - Handling relationships between model types.
  - Setup node services and server-side rendering on the same process.
  - How to turn off parts of the app that should not be server-side rendered.
+
+## [Contributing](./contributing.html)
```

Comment:
Is that how DocumentJS works or is that only in `bit-docs`? Note that I want to merge this to `master`.